ledger-api/grpc-definitions/com/daml/ledger/api/v1/version_service.proto

```diff
@@ -13,6 +13,7 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 service VersionService {
 
   // Read the Ledger API version
+  // May return ``INTERNAL`` gRPC error in case of internal configuration errors of the Node.
```

Comment:
```suggestion
  // May return ``INTERNAL`` in case of unexpected internal errors.
```

Reply:
Isn't that true for most services though?

Reply:
I need to check what is the default failure error code because it might be `UNKNOWN` as well. In this particular service all failures will end up with `INTERNAL`.